### PR TITLE
JPERF-969: flaky async profiler

### PR DESCRIPTION
**Pull request by dagguh**

### CODE REVIEW

**Comment by dagguh** [comment]
Turns out, even after the bump, the flakiness is still there.
At least now we've got a 40 second dev loop here.

**Comment by dagguh** [comment]
PS. now it tests: install jira, install profiler, start jira, start profiler.
Beforehand the test was: install jira, start jira, install profiler, start profiler. And it wasn't reproducing then.
Perhaps it's about the time since Jira start (JVM start).

**Comment by dagguh** [comment]
- [ ] harvest the [checksum check](https://stash.atlassian.com/projects/JIRASERVER/repos/jira-performance-tests/pull-requests/1043/commits/c6d24bf5119f7fa7e473067e1c4cac0233dd3864#jira-software-tests/src/test/kotlin/com/atlassian/jira/test/performance/lib/infrastructure/api/profiler/BumpedAndChecksummingAsyncProfiler.kt)

**Comment by dagguh** [comment]
Pushed a rebase onto master to get nicer CI from #136

File: src/main/kotlin/com/atlassian/performance/tools/infrastructure/api/profiler/AsyncProfiler.kt [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+import com.atlassian.performance.tools.jvmtasks.api.StaticBackoff
 import com.atlassian.performance.tools.ssh.api.SshConnection
+import java.time.Duration.ofSeconds
 
 /**
  *  Asynchronous profiler. See https://github.com/jvm-profiling-tools/async-profiler#basic-usage
  */
 class AsyncProfiler : Profiler {
+
+    private val release = "async-profiler-2.9-linux-x64"
```
Comment (mgrzaslewicz) [inline src/main/kotlin/com/atlassian/performance/tools/infrastructure/api/profiler/AsyncProfiler.kt:14]: `profilerVersion`?

Comment (dagguh) [reply src/main/kotlin/com/atlassian/performance/tools/infrastructure/api/profiler/AsyncProfiler.kt:14]: `profilerVersion = "2.9"`
This is a specific release (for 64-bit Linux).

**Review by mgrzaslewicz — APPROVED** [review]

**Pull request merged**